### PR TITLE
Style Personal Health Care Contacts to new designs

### PR DESCRIPTION
**Pull request by radavis**
## Summary

- Style Personal Health Care Contacts to new designs

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#72549

## Testing done

- Tested locally

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |   Screenshot 2024-03-13 at 09-40-38 Personal Health Care Contacts Veterans Affairs    |  Screenshot 2024-03-05 at 12-13-40 Personal Health Care Contacts Veterans Affairs   |
| Desktop |    Screenshot 2024-03-13 at 09-41-00 Personal Health Care Contacts Veterans Affairs    |   Screenshot 2024-03-05 at 12-13-21 Personal Health Care Contacts Veterans Affairs    |

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

File: src/applications/personalization/profile/components/personal-health-care-contacts/HelpDeskContact.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { CONTACTS } from '@department-of-veterans-affairs/component-library/contacts';
+
+const HelpDeskContact = ({ testId }) => (
+  <>
+    <va-telephone
```
Comment (carlosfelix2) [inline src/applications/personalization/profile/components/personal-health-care-contacts/HelpDeskContact.jsx:7]: You may need to also create tests for your new files. I don't see anything testing these.

**Review by carlosfelix2 — CHANGES_REQUESTED** [review]

File: src/applications/personalization/profile/components/personal-health-care-contacts/HelpDeskContact.jsx (hunk 1 above)
Comment (radavis) [reply src/applications/personalization/profile/components/personal-health-care-contacts/HelpDeskContact.jsx:7]: https://github.com/department-of-veterans-affairs/vets-website/blob/radavis/72549-contacts-design-update/src/applications/personalization/profile/components/personal-health-care-contacts/PersonalHealthCareContacts.unit.spec.jsx#L46:L46

File: src/applications/personalization/profile/components/personal-health-care-contacts/Contact.jsx [hunk 2]
```diff
@@ -39,24 +38,34 @@ const Contact = ({
     addressLine3,
     addressLine4,
   ].filter(line => !!line);
-  const showAddress = contactType.match(/next of kin/i);
+  const isNextOfKin = contactType.match(/next of kin/i);
+  let title = index === 0 ? 'Primary' : 'Secondary';
```
[the remark this answers is not in the document]
Comment (carlosfelix2) [reply src/applications/personalization/profile/components/personal-health-care-contacts/Contact.jsx:42]: Covered in new changes to unit tests

**Review by jzucadi — CHANGES_REQUESTED** [review]

**Review by carlosfelix2 — DISMISSED** [review]

File: src/applications/personalization/profile/components/personal-health-care-contacts/Contact.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (radavis) [reply src/applications/personalization/profile/components/personal-health-care-contacts/Contact.jsx:42]: I was trying no to put so much logic into one line

**Review by jzucadi — DISMISSED** [review]

**Review by allisonlu — COMMENTED** [review]
lgtm on the UI side :shipit: but i'll let @adamwhitlock1 approve on behalf of AE FE

the only thing i see that may be a concern is `{description}` may be redundant, but i'm no content expert

File: src/applications/personalization/profile/components/personal-health-care-contacts/Contacts.jsx [hunk 3]
```diff
@@ -86,27 +67,18 @@ const Contacts = ({ data }) => {
...
-          <p className="vads-u-color--gray-medium vads-u-margin-top--1 vads-u-margin-bottom--1">
-            The people we’ll contact in an emergency.
-          </p>
-          {renderEmergencyContacts}
-        </div>
+      <ProfileInfoCard
+        title="Emergency contacts"
+        data={renderEmergencyContacts}
+        namedAnchor="emergency-contacts"
+      />
```
Comment (adamwhitlock1) [inline src/applications/personalization/profile/components/personal-health-care-contacts/Contacts.jsx:74]: I was looking at the heading map, and I _think_ the title for each card should be set to an h2 and then the title of each contact can stay as an h3. The `ProfileInfoCard` has a level prop that should suffice for this.

Right now the heading map looks like this:
Screenshot 2024-03-05 at 9 13 37 AM

vs the other pages will usually look more like this:
Screenshot 2024-03-05 at 9 13 54 AM

side not: we do recognize the issue with the profile menu heading, and there has been talk about fixing that heading mismatch, but it is widespread throughout all the va pages with a sidebar, and hasn't been solved yet, but the design system team is working on that with content/IA folks.

**Comment by adamwhitlock1** [comment]
I saw the reference to the design file in the linked ticket show this:

design for NOK EC

This looks like the description of each contact "The person we'll contact in an emergency" should be closer to the heading inside each card. Should we reduce the margin on those <p> tags?

**Review by carlosfelix2 — APPROVED** [review]

**Review by adamwhitlock1 — APPROVED** [review]
Everything is looking good in the newest commit. Great job @radavis!

**Review by jzucadi — APPROVED** [review]

**Pull request merged**